bench/wtperf/config.c

```diff
@@ -558,7 +560,7 @@ config_opt_file(CONFIG *cfg, const char *filename)
 		goto err;
 	}
 	buf_size = (size_t)sb.st_size;
-	file_buf = dcalloc(buf_size + 2, 1);
+	file_buf = dmalloc(buf_size + 2);
```

Comment:
@keithbostic, it looks like this could be one less byte: `dmalloc(buf_size + 1)`